index.html

```diff
@@ -637,6 +637,76 @@ <h2><b>2. TEXT AREA:-</b></h2>
     &lt;option value="cycle"&gt;cycle&lt;/option&gt;<br>
     &lt;/select&gt;<br>
     </p>
+    <h2>ATTRIBUTES for select</h2>
+    <ol>
+      <li>Name:- Adds an internal name of the field</li>
+      <li>Multiply:- Allows multiple choices if yes.</li>
+      <li>Option:- Individual items in the menu.</li>
+      <li>Size:- Use the size attribute to specify the number of visible values.</li>
+    </ol>
+    <h2>ATTRIBUTES FOR OPTIONS</h2>
+    <ol>
+      <li>Selected:- Default select the item</li>
+      <li>Value:- Value to send if selected </li>
+    </ol>
+    
+    <p style="text-align:center">Page 11</p>
+  </section>
+  <section class="page">
+    <h3>4.&lt;fieldset&gt; and &lt;legend&gt; Elements:-</h3>
+    <ul>
+      <li>The &lt;fieldset&gt; element is used to group related data in a form.</li>
+      <li>The &lt;legend&gt;elements defines a caption for the &lt;fieldset&gt; element. </li>
+    </ul>
+    <h3>5. &lt;datalist&gt; Element</h3>
+    <ul>
+      <li>The &lt;datalist&gt; element specifies a list of pre-defined options for an &lt;input&gt; elements </li>
+      <li>User will see a drop -down list of the pre-defined options as they input data.</li>
+    </ul>
+    <p class="datalist"><br>
```

Comment:
please use code tag instead of p while writing code do this change in whole file